docs/content/getting-started/perform-list-users.mdx

```diff
@@ -155,7 +155,7 @@ To return all users of type `user` that have have the `reader` relationship with
   relation="reader"
   userFilterType="user"
   expectedResults={{
-    users: [{ object: { type: "user:anne" } }, { object: { type: "user:beth" } }],
+    users: [{ object: { type: "user", id: "anne" } }, { object: { type: "user", id: "beth" } }],
```

Comment:
Again, not related to your PR, can you fix this typo?

![Uploading image.png…]()